src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -67,7 +67,7 @@ def __call__(
         prompt: Union[str, List[str]],
         init_image: Union[torch.FloatTensor, PIL.Image.Image],
         mask_image: Union[torch.FloatTensor, PIL.Image.Image],
-        strength: float = 0.8,
+        strength: float = 0.0,
```

Comment:
No, that's the opposite, it should be 1.
With strength 0 there is only one iteration.